README.md

```diff
@@ -89,7 +89,7 @@ to install fzf.
 brew install fzf
 
 # To install useful key bindings and fuzzy completion:
-$(brew --prefix)/opt/fzf/install
+"$(brew --prefix)/opt/fzf/install"
```

Comment:
I'm unsure about this one :thinking:

Reply:
The brew prefix path might include spaces, so strictly speaking it _must_ be quoted (no one ever uses a non-default brew prefix, so yeah..)

Reply:
Makes sense!